benchmarks/Makefile.am

```diff
@@ -1,15 +1,27 @@
 AM_CPPFLAGS = -I$(top_srcdir)/include @nfft_benchmarks_CPPFLAGS@ @CPPFLAGS@
 
 if HAVE_BENCHMARKS
-  NFFT_BENCHMARKS = bench_nfft_direct
+if BENCHMARK_AGNOSTIC_WINDOW
```

Comment:
If I haven't overlooked something, this seems to be the only time we use this flag. Do the others (like BENCHMARK_AGNOSTIC_PRECISION) have an effect right now?

Reply:
No, not at the moment. I was thinking ahead when we are adding more benchmarks.